src/entities/errors.ts

```diff
@@ -36,10 +36,16 @@ export enum ErrorCode {
   CustomerInfoError = 28,
   SignatureVerificationError = 36,
   InvalidEmailError = 38,
+  RateLimitError = 39,
+  TaxCollectionNotEnabledError = 40,
+  TaxLocationCannotBeDeterminedError = 41,
+  InvalidTaxLocationError = 42,
+  GatewaySetupError = 43,
+  SandboxModeOnlyError = 44,
 }
 
 export class ErrorCodeUtils {
-  static getPublicMessage(errorCode: ErrorCode): string {
+  static getPublicMessage(errorCode: ErrorCode): string | undefined {
```

Comment:
Not every ErrorCode should have a public message. Hence the `undefined`

Reply:
I wonder about this... I think these developer facing error codes should all have a message. In what situation we wouldn't want to have a message?

Reply:
Oh, I thought `PublicMessage` referred to an end-customer-facing message. I also found that `PurchasesError.message` can be `undefined`, so I thought that might be the case as well.